runtime/php-script-globals.cpp

```diff
@@ -6,6 +6,19 @@
 
 static PhpScriptMutableGlobals php_script_mutable_globals_singleton;
 
+// actually, g_linear_mem is allocated and never freed (since workers live forever),
+// but to prevent leak sanitizer errors, free memory manually (descructor is called of worker's stop)
+PhpScriptMutableGlobals::~PhpScriptMutableGlobals() {
+  if (g_linear_mem != nullptr) {
+    delete g_linear_mem;
+    g_linear_mem = nullptr;
+  }
+  for (const auto &[k, _] : libs_linear_mem) {
+    delete libs_linear_mem[k];
```

Comment:
```c++
for (const auto &[_, lib_mem] : libs_linear_mem) {
  delete lib_mem;
```

Reply:
At first, I've written exactly the same code, but CLion (or clang-tidy?) highlights `delete lib_mem` as a warning. Maybe, it's false positive, but nevertheless, I've rewritten in that way

Reply:
> At first, I've written exactly the same code, but CLion (or clang-tidy?) highlights `delete lib_mem` as a warning. Maybe, it's false positive, but nevertheless, I've rewritten in that way

Oh, I suppose it's the problem with const- and reference-ness of structured binding.
Just `for (auto [_, lib_mem] : libs_linear_mem)` will be without warnings I believe.